### PR TITLE
Beta parameters plugin

### PR DESCRIPTION
**Pull request by TaylorKrusen**
PR for adding beta plugin

### CODE REVIEW

**Comment by TaylorKrusen** [comment]
Note: hold merge for specific timing

File: bin/plugins/beta-fields-plugin.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const decorators = {
+    oas3: {
+        'check-request-schema-for-beta': addBetaToFields,
+    },
+};
+
+module.exports = {
+    id,
+    decorators,
+};
```
Comment (nboricic) [inline bin/plugins/beta-fields-plugin.js:14]: New line at the end of file

File: translations/en-plugin.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "betaClosed": {
+        "disclaimer": "Only available in closed beta.",
+        "emailLink": "Contact us",
+        "disclaimerContinued": "to request access.",
+        "subjectForEmail": "Requesting access to beta HelloSign feature"
+    }
+}
```
Comment (nboricic) [inline translations/en-plugin.json:8]: Newline at the end of file

File: bin/plugins/decorators/check-beta-fields.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+*/
+
+// Helper function
+const addClosedBetaCopy = (description) => {
+    const { betaClosed } = englishStrings
+    const descriptionWithBeta = 
+        description +
+        "\n" +
+        `<div class='beta-closed-field-description'>${betaClosed.disclaimer} <a href='mailto:apisupport@hellosign.com?subject=${betaClosed.subjectForEmail}'>${betaClosed.emailLink}</a> ${betaClosed.disclaimerContinued}</div>`;
+        "<div class='beta-closed-field-description'>This parameter isn't available quite yet. <a href='apisupport@hellosign.com?subject=Request for Beta feature'>Contact us</a> if you'd like to test the beta version.</div>";
```
Comment (nboricic) [inline bin/plugins/decorators/check-beta-fields.js:16]: Are we not able to have this translated?

**Review by nboricic — APPROVED** [review]

**Pull request merged**